ui/src/views/Tasks/routes.js

```diff
@@ -72,6 +72,14 @@ export default path => [
     component: TaskLog,
     path: `${path}/:taskId/runs/:runId/logs/:logUrl`,
   },
+  {
+    component: TaskLog,
+    path: `${path}/:taskId/runs/:runId/log/:artifactFileName`,
+  },
+  {
+    component: TaskLog,
+    path: `${path}/:taskId/runs/:runId/log/live/:artifactFileName`,
+  },
```

Comment:
Once the TaskLog page is loaded, you will need to:

1. Extract the name, run ID, and task ID use it to figure out the actual URL of the log
2. Feed the URL from step 1 to the log viewer `url` prop

This PR does not do any of that logic yet so perhaps that's why you are not seeing the log.